.github/workflows/conda.yml

```diff
@@ -140,4 +140,64 @@ jobs:
         run: |
           # install anaconda for upload
           mamba install anaconda-client
-          anaconda upload dist/conda/noarch/*.tar.bz2
\ No newline at end of file
+          anaconda upload dist/conda/noarch/*.tar.bz2
+
+  report:
+    name: report
+    needs: conda
+    if: |
+      always()
+      && github.event_name == 'pull_request'
+      && github.repository == 'coiled/coiled-runtime'
+      && needs.conda.result == 'failure'
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v2
+      - name: Report failures
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const title = "⚠️ Main CI failed ⚠️"
```

Comment:
Nit: Even though this has less content, it somehow seems more clear to me. What do you think? 

```suggestion
            const title = "⚠️ CI failed ⚠️"
```